apis/database/v1beta1/dbsubnetgroup_types.go

```diff
@@ -162,7 +154,7 @@ func (b *DBSubnetGroup) UpdateExternalStatus(observation rds.DBSubnetGroup) {
 		}
```

Comment:
I think the fields in spec and status need some changes. You can take a look at our API patterns doc to get a grasp of our conventions around the fields https://github.com/crossplane/crossplane/blob/master/design/one-pager-managed-resource-api-design.md#high-fidelity